presto-docs/src/main/sphinx/functions/string.rst

```diff
@@ -168,6 +168,28 @@ String Functions
     position ``start``. Positions start with ``1``. A negative starting
     position is interpreted as being relative to the end of the string.
 
+.. function:: translate(source, from, to) -> varchar
+
+   Returns the ``source`` string translated by replacing characters found in the
+   ``from`` string with the corresponding characters in the ``to`` string.  If the ``from``
+   string contains duplicates, only the first is used.  If the ``source`` character
+   does not exist in the ``from`` string, the ``source`` character will be copied
+   without translation.  If the index of the matching character in the ``from``
+   string is beyond the length of the ``to`` string, the ``source`` character will
+   be omitted from the resulting string.
+
+   Here are some examples illustrating the translate function::
+
+       translate('abcd', '', '') -- 'abcd'
+       translate('abcd', 'a', 'z') -- 'zbcd'
+       translate('abcda', 'a', 'z') -- 'zbcdz'
+       translate('Palhoça', 'ç','c') -- 'Palhoca'
+       translate('abcd', 'b', '\uD840\uDC00') -- 'a\uD840\uDC00cd'
```

Comment:
This `\u` is actually a Java escape and not valid in SQL (it works in the test because it's translated by the Java compiler, so it's effectively a literal character in the SQL)

We can write this using the SQL `U&` syntax as documented here: https://prestosql.io/docs/current/language/types.html#string

For the output, we can paste the literal in directly.
```sql
SELECT translate('abcd', 'b', U&'\+01F600'); -- a😀cd